pontoon/localizations/views.py

```diff
@@ -108,6 +125,28 @@ def ajax_resources(request, code, slug):
     })
 
 
+@require_AJAX
+def ajax_tags(request, code, slug):
+    """Tags tab."""
+    locale = get_object_or_404(Locale, code=code)
+    project = get_object_or_404(Project, slug=slug)
+
+    if not project.tags_enabled:
```

Comment:
[nit] Class-based views could help generalize that logic and move it into one place.

Reply:
I'll leave that refactor for later, we might have more related views coming.